spec.html

```diff
@@ -43572,7 +43572,7 @@ <h1>PromiseCapability Records</h1>
                 [[Promise]]
               </td>
               <td>
-                an Object
+                an Object or *undefined*
               </td>
               <td>
                 An object that is usable as a promise.
```

Comment:
If we *did* change these types, we would have to update the descriptions to indicate what `*undefined*` *meant*, but it has no meaning, so I don't expect we'll do this.